File: data/translated_projects/commons-fileupload/src/main/org/apache/commons/fileupload/util/Closeable.py
from __future__ import annotations
from abc import ABC
import io
import os
import java # type: ignore
from src.main.org.apache.commons.fileupload.java_handler import *
class Closeable(ABC, metaclass=StaticFieldRedirector):
    def isClosed(self) -> bool:
        pass


    def close(self) -> None:
        pass


    javaClz = java.type("org.apache.commons.fileupload.util.Closeable")
    def __getattr__(self, name):
        try:
            return object.__getattribute__(self, name)
        except AttributeError:
            pass
        return JavaHandler.mapping(getattr(self.javaClz, StaticFieldRedirector.unmangle_name(name)))

File: logs/glue/commons-fileupload/src/main/java/org/apache/commons/fileupload/util/Closeable.java
package org.apache.commons.fileupload.util;

import java.io.IOException;
import org.apache.commons.fileupload.ContextInitializer;
import org.apache.commons.fileupload.ExceptionHandler;
import org.apache.commons.fileupload.IntegrationUtils;
import org.graalvm.polyglot.PolyglotException;
import org.graalvm.polyglot.Value;

public interface Closeable {
  boolean isClosed() throws IOException;

  void close() throws IOException;
}
